Containers/talk-recording/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN set -ex; \
     echo "root:$(openssl rand -base64 12)" | chpasswd; \
     git clone --recursive https://github.com/nextcloud/spreed --depth=1 --single-branch --branch "$RECORDING_VERSION" /src; \
     mv -v /src/recording/pyproject.toml /src/recording/src/pyproject.toml; \
-    python3 -m pip install --no-cache-dir /src/recording/src; \
+    python3 -m pip install --no-cache-dir --no-index --find-links=file:///local/dir /src/recording/src; \
```

Comment:
I guess I need to adjust `file:///local/dir` ? question is to what but I suppose that is what I need to figure out myself 😅

Reply:
Current dir I guess?

Reply:
Speaking of which, the recording backend was moved to it's own repo:
https://github.com/nextcloud/nextcloud-talk-recording

Reply:
right, thanks for the heads-up!

Reply:
https://github.com/nextcloud/all-in-one/issues/3570

Reply:
Actuallly we are installing the packages from the pyproject file so I am not sure we actually need the above code changes here? @danxuliu please clarify :)

Also see https://github.com/nextcloud/nextcloud-talk-recording/blob/main/pyproject.toml

Reply:
Using `--no-index` disables getting the packages from PyPi. However, it does it not only for the package being installed, but also its dependencies.

However, to locally install a package it is enough to use an identifier that hints at being a local directory. So installing from `/src/recording/src`* should be fine; to err on the side of caution you could prepend `file://`, but it should not be necessary.

[The problem in the recording server documentation](https://github.com/nextcloud/nextcloud-talk-recording/pull/7) was that it was installed from a relative directory named `nextcloud-talk-recording`, so pip tried to fetch it from PyPi too.

*Out of curiosity, why do you move the `pyproject.toml` file inside the `src` directory? It should be enough to keep it in the original place and install `/src/recording`, or what am I missing? :-)